Continuous error stack trace after starting the enforcer component - Choreo Connect 1.0.0

### Description:
Continuous error stack trace after starting the enforcer component

### Steps to reproduce:

- Start the API Manager 4.0.0 pack (U2 level: 107 latest)
- Execute the docker-compose command to start all the Choreo Connect components and able to see the below error stack trace.

```
enforcer_1  | [2022-05-19 14:46:37,050][] ERROR - {org.wso2.choreo.connect.enforcer.discovery.ConfigDiscoveryClient} - Error occurred during Config discovery
enforcer_1  | io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
enforcer_1  | Channel Pipeline: [SslHandler#0, ProtocolNegotiators$ClientTlsHandler#0, WriteBufferingAndExceptionHandler#0, DefaultChannelPipeline$TailContext#0]
enforcer_1  |   at io.grpc.stub.ClientCalls.toStatusRuntimeException(ClientCalls.java:262) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:243) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:156) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at org.wso2.choreo.connect.discovery.service.config.ConfigDiscoveryServiceGrpc$ConfigDiscoveryServiceBlockingStub.fetchConfigs(ConfigDiscoveryServiceGrpc.java:234) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at org.wso2.choreo.connect.enforcer.discovery.ConfigDiscoveryClient.requestInitConfig(ConfigDiscoveryClient.java:116) [org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at org.wso2.choreo.connect.enforcer.discovery.ConfigDiscoveryClient.run(ConfigDiscoveryClient.java:138) [org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:?]
enforcer_1  |   at java.util.concurrent.FutureTask.runAndReset(Unknown Source) [?:?]
enforcer_1  |   at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:?]
enforcer_1  |   at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:?]
enforcer_1  |   at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:?]
enforcer_1  |   at java.lang.Thread.run(Unknown Source) [?:?]
enforcer_1  | Caused by: io.grpc.netty.shaded.io.netty.handler.ssl.SslHandshakeTimeoutException: handshake timed out after 10000ms
enforcer_1  |   at io.grpc.netty.shaded.io.netty.handler.ssl.SslHandler$5.run(SslHandler.java:2062) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at io.grpc.netty.shaded.io.netty.util.concurrent.PromiseTask.runTask(PromiseTask.java:98) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at io.grpc.netty.shaded.io.netty.util.concurrent.ScheduledFutureTask.run(ScheduledFutureTask.java:170) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at io.grpc.netty.shaded.io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at io.grpc.netty.shaded.io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at io.grpc.netty.shaded.io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:384) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at io.grpc.netty.shaded.io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at io.grpc.netty.shaded.io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   at io.grpc.netty.shaded.io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[org.wso2.choreo.connect.enforcer-1.0.0.jar:1.0.0]
enforcer_1  |   ... 1 more
```

### Affected Product Version:
1.0.0

--- a/adapter/internal/adapter/adapter.go
+++ b/adapter/internal/adapter/adapter.go
@@ -358,6 +358,10 @@ func fetchAPIsOnStartUp(conf *config.Config, apiUUIDList []string) {
 				})
 			}
 			health.SetControlPlaneRestAPIStatus(err == nil)
+		} else if data.ErrorCode == 204 {
+          			logger.LoggerMgw.Infof("No API Artifacts are available in the control plane for the envionments :%s",
+          				strings.Join(envs, ", "))
+          			health.SetControlPlaneRestAPIStatus(true)
 		} else if data.ErrorCode >= 400 && data.ErrorCode < 500 {
 			logger.LoggerMgw.ErrorC(logging.ErrorDetails{
 				Message:   fmt.Sprintf("Error occurred when retrieving APIs from control plane(unrecoverable error): %v", data.Err.Error()),

--- a/adapter/pkg/synchronizer/apis_fetcher.go
+++ b/adapter/pkg/synchronizer/apis_fetcher.go
@@ -155,7 +155,7 @@ func FetchAPIs(id *string, gwLabel []string, c chan SyncAPIResponse, serviceURL 
 		req.Header.Set("Content-Type", "application/json")
 	}
 	// Make the request
-	logger.LoggerSync.Debug("Sending the controle plane request")
+	logger.LoggerSync.Debug("Sending the control plane request")
 	resp, err = client.Do(req)
 	// In the event of a connection error, the error would not be nil, then return the error
 	// If the error is not null, proceed
@@ -188,7 +188,6 @@ func FetchAPIs(id *string, gwLabel []string, c chan SyncAPIResponse, serviceURL 
 	}
 	// If the response is not successful, create a new error with the response and log it and return
 	// Ex: for 401 scenarios, 403 scenarios.
-	logger.LoggerSync.Errorf("Failure response: %v", string(respBytes))
 	respSyncAPI.Err = errors.New(string(respBytes))
 	respSyncAPI.Resp = nil
 	respSyncAPI.ErrorCode = resp.StatusCode

